Microchip/module.mk

```diff
@@ -1,5 +1,7 @@
 # this file is included from makefile
 
+ifneq (,$(filter $(TOOLCHAIN), C30 XC16))
+
 extra_dirs := MDD-File-System USB USB/CDC-Device-Driver USB/MSD-Device-Driver

```

Comment:
I've semi started with moving over to the FatFs library to replace the Microchip library code. Unfortunately as of this moment, it's a lower priority than that which draws my motivation. Also, i'm strongly in favour of keeping the Microchip filesystem library for the AUAV3 bootloader - so perhaps for that particular sub project, we could re-licence it as something akin the the LGPL.